### ISSUE TITLE
Loss calculation 

### ISSUE THREAD

**Issue by zhouxin666251** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#161]
When calculating the loss, the size of the picture is 46x46. If the size of the output image is 23x23, how should it be calculated? Does the following method work? 

keypoint_maps = batch_data['keypoint_maps'].cuda()
keypoints_maps=keypoint_maps/2

**Comment by Daniil-Osokin** [comment]
Hi! How output image size became 23x23 instead of 46x46? If you have just changed [crop size](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/57db93e3b7254acaa0f03bb67950738ff7d7b1a6/datasets/transformations.py#L132), the output should be changed correspondingly automatically.

**Comment by zhouxin666251** [comment]
Change the stridee of a certain convolution kernel from 1 to 2 ?

**Comment by Daniil-Osokin** [comment]
In this case, set the [stride](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/train.py#L28) to 16.

**Comment by zhouxin666251** [comment]
thanks, have a nice day
